### PR TITLE
Set  mounting parameters according to the Buildah version when building Containers

### PR DESCRIPTION
**Pull request by Sinerum**
Set the `rw` mounting parameter when the installed buildah version requires it and not if it does not. Also update the required benchbuild version to 3.6.1. to include this fix in benchbuild as well.

### CODE REVIEW

**Review by boehmseb — APPROVED** [review]

**Comment by boehmseb** [comment]
Only this one test needs fixing. It just needs to include the `,rw` in the expected output.

**Comment by Sinerum** [comment]
> Only this one test needs fixing. It just needs to include the `,rw` in the expected output.

It needs to be removed, but that would mean the test would fail depending on the buildah version would that be okay?

File: varats/varats/containers/containers.py [hunk 1]
```diff
@@ -60,7 +61,7 @@ def distro(self) -> Distro:
             'apt', 'install', '-y', 'wget', 'gnupg', 'lsb-release',
             'software-properties-common', 'python3', 'python3-dev',
             'python3-pip', 'musl-dev', 'git', 'gcc', 'libgit2-dev',
-            'libffi-dev', 'libyaml-dev'
+            'libffi-dev', 'libyaml-dev', 'graphviz-dev'
```
Comment (boehmseb) [inline varats/varats/containers/containers.py:64]: Can you please also add this dependency to the install guide in `docs/source/tutorials/getting_started.rst` and `docs/source/vara-ts/vara-buildsetup.rst`?

**Review by boehmseb — CHANGES_REQUESTED** [review]

**Comment by vulder** [comment]
> > Only this one test needs fixing. It just needs to include the `,rw` in the expected output.
> 
> It needs to be removed, but that would mean the test would fail depending on the buildah version would that be okay?

Neither, just use the buildah version check to determine if `rm` should be there or not.

**Review by vulder — APPROVED** [review]
Except for the missing test adaption this PR looks good to me.

**Pull request merged**